gemrb/plugins/GUIScript/PythonHelpers.cpp

```diff
@@ -111,3 +75,62 @@ void PythonControlCallback::operator() (Control* ctrl) const
 	Py_DECREF(co_argcount);
 	CallPython(Function, args);
 }
+
+template <>
+bool PythonObjectCallback<WindowKeyPress>::operator() (WindowKeyPress *wkp) {
```

Comment:
This too ought to already be handled by the existing python callbacks. The python representation of the window (`GWindow`) would be passed as the parameter to the python function.